pmsv2/card/params.go

```diff
@@ -0,0 +1,75 @@
+package card
+
+import (
+	"github.com/xendit/xendit-go/pmsv2/constant"
+)
+
+type CardOnFileType string
+
+const (
+	CustomerUnscheduledCOF CardOnFileType = "CUSTOMER_UNSCHEDULED"
+	MerchantUnscheduledCOF CardOnFileType = "MERCHANT_UNSCHEDULED"
+	RecurringCOF           CardOnFileType = "RECURRING"
+)
+
+type CreateMethod struct {
+	Currency          *constant.CurrencyEnum `json:"currency"`
+	ChannelProperties *CardChannelProperties `json:"channel_properties"`
+	Token             *Token                 `json:"token"`
```

Comment:
remove `Token`

Reply:
you think we should include `card_information` here?

Reply:
told ligert what to add